app/scripts/Annotations1dTrack.js

```diff
@@ -164,13 +177,44 @@ class Annotations1dTrack extends PixiTrack {
         );
       }
 
+      // Make annotation clickable
+      this.rects[id].graphics.clear();
+      this.rects[id].graphics.interactive = true;
+      this.rects[id].graphics.buttonMode = true;
+
       graphics.beginFill(fillHex, +region[4] || globalFillOpacity);
       if (this.isVertical) {
         graphics.drawRect(0, start, this.dimensions[0], width);
+        this.rects[id].graphics.hitArea = new PIXI.Rectangle(
+          0, start, this.dimensions[0], width
+        );
       } else {
         graphics.drawRect(start, 0, width, this.dimensions[1]);
+        this.rects[id].graphics.hitArea = new PIXI.Rectangle(
+          start, 0, width, this.dimensions[1]
+        );
       }
+
+      this.rects[id].graphics.mousedown = () => {
+        this.rects[id].mouseDownTime = performance.now();
+      };
+
+      this.rects[id].graphics.mouseup = (event) => {
+        if (performance.now() - this.rects[id].mouseDownTime < MAX_CLICK_DELAY) {
```

Comment:
Why is this necessary? Are you trying to distinguish from dragging?

Reply:
I though I replied but maybe I didn't.

This is used for distinguishing between a canceled click (long click) and a normal click